bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -1784,6 +1784,49 @@ Mapping method from Nitrogen deposition input file to the model resolution
     copy     = copy using the same indices
 </entry>
 
+<!-- ========================================================================================  -->
+<!-- Dust namelist and Zender soil erodibility stream                                          -->
+<!-- ========================================================================================  -->
+
+<entry id="dust_emis_method" type="char*12" category="physics"
+       group="clm_inparm" valid_values="Zender_2003,Leung_2023" >
+Which dust emission method is going to be used. Either the Zender 2003 scheme or the Leung 2023
+scheme.
+(NOTE: The Leung 2023 method is NOT currently available)
+</entry>
+
+<entry id="zender_soil_erod_source" type="char*3" category="physics"
+       group="zendersoilerod" valid_values="lnd,atm" >
+Option only applying for the Zender_2003 method for whether the soil erodibility file is handled
+here in CTSM, or in the ATM model.
+(only used when dust_emis_method is Zender_2003)
+</entry>
```

Comment:
Erik pointed out to me that the two options above will be moved to cmeps.

Reply:
Here's the PR for that. https://github.com/ESCOMP/CMEPS/pull/429